### PR TITLE
Bump cardano-node to 1.13.0

### PR DESCRIPTION
**Pull request by rvl**
### Overview

Updates to latest cardano-node release [1.13.0](https://github.com/input-output-hk/cardano-node/releases/tag/1.13.0).

### CODE REVIEW

File: nix/sources.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
     "cardano-node": {
-        "branch": "master",
+        "branch": "1.13.0",
```
Comment (Anviking) [inline nix/sources.json:3]: Also, we shouldn't forget to update the readme matrix!

**Review by Anviking — APPROVED** [review]

File: nix/sources.json (hunk 1 above)
Comment (rvl) [reply nix/sources.json:3]: Thanks, fixed.

**Comment by rvl** [comment]
bors try

**Comment by rvl** [comment]
bors try

File: lib/shelley/test/data/cardano-node-shelley/start.sh [hunk 2]
```diff
@@ -1,8 +1,16 @@
 #!/usr/bin/env bash
 
-startTime=$(gdate --iso-8601=s --date="5 seconds")
+set -euo pipefail
 
-cat genesis.yaml | yq ".startTime=\"$startTime\"" > tmp-genesis.json
+if type -p gdate > /dev/null; then
+  gnu_date=gdate
```
Comment (Anviking) [inline lib/shelley/test/data/cardano-node-shelley/start.sh:6]: btw, thanks for this!

**Pull request merged**